### ISSUE TITLE
Imports aren't properly documented

### ISSUE THREAD

**Issue by timtroendle** [calliope-project/euro-calliope#230]
Following updates in #145, imports aren't properly documented. File names of the previously existing imports (transmission links) are broken. All new imports aren't mentioned at all.

**Comment by brynpickering** [comment]
What do you mean by `imports`? The contents of `import` in `example-model.yaml`?

**Comment by timtroendle** [comment]
Yes and the import section in the documentation: Model/Customisation/Imports.
